mainapp/templates/account/signup.html

```diff
@@ -22,6 +24,8 @@ <h1>{% trans "Sign Up" %}</h1>
         <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
         {% endif %}
         <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        <a class="indent">
+        <button class="btn btn-primary" onclick="goBack()">Back</button></a>
```

Comment:
![Screen Shot 2019-11-02 at 4 29 34 PM](https://user-images.githubusercontent.com/32490273/68076661-04400a80-fd8e-11e9-91b7-4af0dc77c5c0.png)

As you can see in the screenshot above the Back button is off-center from the Sign up button. I suggest you take a look at how the 3 buttons in the index page (search, sign In, Sign Up) and take inspiration from that.  Pay special attention the the `<div>`s that the buttons are wrapped in and see how they are used to position the buttons in the page.